website/docs/r/is_vpc_dns_resolution_binding.html.markdown

```diff
@@ -0,0 +1,124 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_vpc_dns_resolution_binding"
+description: |-
+  create VPCDNSResolutionBinding
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_vpc_dns_resolution_binding
```

Comment:
Note - This feature is a Select Location availability invitation only feature. If used in other regions may lead to inconsistencies in state management.

Reply:
<img width="717" alt="image" src="https://github.com/ibm-vpc/terraform-provider-ibm/assets/78945437/768d79c8-6e20-4ee0-9c6e-2282d87b9bd0">